schema/causality_jwt.js

```diff
@@ -36,7 +36,9 @@ export default {
       if (options.role === 'OPERATOR' && me.type !== 'Admin') {
         throw new Error('Unauthorized to act as an operator');
       }
-      const registered = find(bidders, (b) => b.sale._id === options.sale_id);
+      const registered = find(bidders, (b) => {
+        return (b.sale._id === options.sale_id || b.sale.id === options.sale_id);
```

Comment:
Thanks!
On May 11, 2016 20:21, "Ash Furrow" <notifications@github.com> wrote:

> In schema/causality_jwt.js
> <https://github.com/artsy/metaphysics/pull/246#discussion_r62949037>:
>
> > @@ -36,7 +36,9 @@ export default {
> >        if (options.role === 'OPERATOR' && me.type !== 'Admin') {
> >          throw new Error('Unauthorized to act as an operator');
> >        }
> > -      const registered = find(bidders, (b) => b.sale._id === options.sale_id);
> > +      const registered = find(bidders, (b) => {
> > +        return (b.sale._id === options.sale_id || b.sale.id === options.sale_id);
>
> Was merged earlier, I'll open an issue.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/artsy/metaphysics/pull/246/files/6d57f048e53e89c57fd04bc8f6e830ee25d450c6#r62949037>
>